### PR TITLE
New C char manipulation functions cheat sheet

### PR DESCRIPTION
**Pull request by lemueldsouza**
IA: https://duck.co/ia/view/c_character_manipulation_functions_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@lemueldsouza Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/c-char-manipulation.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+    "id": "c_char_manipulation_cheat_sheet",
+    "name": "C character manipulation functions cheat sheet",
+    "description": "Cheat Sheet for C character manipulation functions from ctype.h",
+    "metadata": {
+    "sourceName": "gnu.org",
+    "url": "http://www.gnu.org/software/libc/manual/html_mono/libc.html"
+    },
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: nit: the indentation looks a little off in this file

**Comment by moollaza** [comment]
@lemueldsouza Thanks! This LGTM :+1:

**Comment by lemueldsouza** [comment]
Oops, I just seen this, should I still change it or are we good?

File: share/goodie/cheat_sheets/json/c-char-manipulation.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
+    "id": "c_char_manipulation_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: @lemueldsouza I just realized your filename and `id` don't match your IA Page

Comment (moollaza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: Please make sure they do!

Comment (moollaza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: I've changed the `id` of your IA Page to match the filename and `id` here

File: share/goodie/cheat_sheets/json/c-char-manipulation.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    "name": "C character manipulation functions cheat sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: This should be title-cased and should not mention `cheat sheet`:

`C Character Manipulations`

Comment (lemueldsouza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: @moollaza

So as far as I understood, 3 changes are needed here:
1. Change the id of IA page
You said you've done it, but I don't see it at this link (https://duck.co/ia/view/c_character_manipulation_cheat_sheet)
Let me know if I should change the file names and send a pull request again or its taken care of.

2. Indentation
3. Title cased "name" field
Should I make these changes and send another pull request? 

I'm little confused here and don't want to do the wrong change so just cross checking.

Comment (moollaza) [inline share/goodie/cheat_sheets/json/c-char-manipulation.json]: > Change the id of IA page You said you've done it, but I don't see it at this link (https://duck.co/ia/view/c_character_manipulation_cheat_sheet) Let me know if I should change the file names and send a pull request again or its taken care of.

I have corrected it now and updated the link above. You'll need to add aliases for "c character manipulation", "c char manipulation functions", etc. The default trigger comes from the filename.

> Indentation

:+1: 

> Title cased "name" field Should I make these changes and send another pull request?

:+1:

**Pull request merged**